Sources/Sentry/include/SentryHttpInterceptor+Private.h

```diff
@@ -0,0 +1,16 @@
+#include "SentryHttpInterceptor.h"
+
+NS_ASSUME_NONNULL_BEGIN
+static NSString *const SENTRY_INTERCEPTED_REQUEST = @"SENTRY_INTERCEPTED_REQUEST";
+
+@interface SentryHttpInterceptor (private) <NSURLSessionDelegate, NSURLSessionTaskDelegate,
+    NSURLSessionDataDelegate>
+
+@property (nullable, nonatomic, strong) NSURLSession *session;
+
++ (void)configureSessionConfiguration:(NSURLSessionConfiguration *)configuration;
+
+- (NSURLSession *)createSession;
```

Comment:
We need this private header to expose SentryHttpInterceptor protocols to tests and createSession must be overwritten in a Test Class couldn't do this with dynamic.

Since i need this file, is more convenient to expose some other things to test instead of using Dynamic.